site_scons/prereq_tools/base.py

```diff
@@ -721,6 +721,7 @@ def define(self, name, **kw):
             extra_include_path -- Subdirectories to add to dependent component path
             out_of_src_build -- Build from a different directory if set to True
             build_env -- Environment variables to set for build
+            skip_arch -- not required on this architecture
```

Comment:
I'm not convinced by the name of this flag, there are many reasons why components might define something to be empty.